planet/src/main/webapp/js/search-bar.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function getSearchResults() {
+    const resultsContainer = document.getElementById('search-results-container');
+    resultsContainer.innerHTML = '';
+    const searchInput = document.getElementById('search-input').value;
+    if (searchInput === '') {
+        return;
+    }
+    database.ref('users').orderByChild('name')  // Order elements by name
+    .startAt(searchInput + 'A') // Start at the users whose names start with searchInput + 'A'
```

Comment:
are there any non-alphabetic characters in usernames? If so, you might not catch them here

Reply:
True! The only non-alphabetic characters in usernames are the space character! I added more characters in the next PR. Should I also add them here?

Reply:
Is space is ordered lexicographically between 'A' and 'z'? I see that it does between those values in the ASCII and Unicode specs, but I don't know where it sorts lexicographically. Have you confirmed this works correctly with spaces?

You can update this logic either in this PR or in the next one to handle other values.

Reply:
sounds good.